docs/using/selecting.md

```diff
@@ -221,10 +221,10 @@ Every Monday and whenever a pull request is merged, images are rebuilt and pushe
 Whenever a docker image is pushed to the container registry, it is tagged with:
 
 - a `latest` tag
-- a 12-character git commit SHA like `85f615d5cafa`
-- a date formatted like `2022-08-04`
+- a 12-character git commit SHA like `1ffe43816ba9`
+- a date formatted like `2023-01-30`
 - OS version like `ubuntu-22.04`
-- a set of software version tags like `python-3.10.5` and `lab-3.4.4`
+- a set of software version tags like `python-3.10.8` and `lab-3.5.3`
```

Comment:
I chose another date here on purpose.
Because when I decide to update the date example next time, this section won't change (and will be consistent).